src/Backend/Modules/ContentBlocks/Command/CopyContentBlocksToOtherLocaleHandler.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Backend\Modules\ContentBlocks\Command;
+
+use Backend\Core\Engine\Model;
+use Backend\Modules\ContentBlocks\Entity\ContentBlock;
+use Backend\Modules\ContentBlocks\Repository\ContentBlockRepository;
+
+final class CopyContentBlocksToOtherLocaleHandler
```

Comment:
This `CopyContentBlocksToOtherLocaleHandler` class is missing an event being dispatched.